frontend/src/components/Simulator/NodeLogs.vue

```diff
@@ -4,13 +4,22 @@ import { webSocketClient } from '@/utils'
 import { useMainStore } from '@/stores';
 
 const scrollContainer = ref<Element>()
+const colorMap: Record<string, string> = {
+  'info': 'text-blue-500',
+  'error': 'text-red-500',
+  'warning': 'text-yellow-500',
+  'success': 'text-green-500'
+}
 
-
+const getDataText = (data: any) => {
+  if (typeof data !== 'string') return JSON.stringify(data)
```

Comment:
@bradleySuira what if we do `JSON.stringify(data, null, 2)` to print it a little bit better?

Reply:
I'll add that